community/deb.sh

```diff
@@ -79,10 +79,19 @@ detect_os ()
   echo "Detected operating system as $os/$dist."
 }
 
+add_pgdg ()
```

Comment:
I think I'd like to verify whether `postgresql-9.5` is available before adding the repository. Looking into how to do this…